CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.22.0 [2023-04-11]
+
+_Dependencies_
+
+- Recompiled plugin with steampipe-plugin-sdk v5.3.0 which adds go-getter support to dynamic tables. ([#496](https://github.com/turbot/steampipe-plugin-oci/pull/496))
```

Comment:
@madhushreeray30 could you please add the changelog link to the steampipe-plugin-sdk v5.3.0 version